node/engine/engine.go

```diff
@@ -657,13 +657,13 @@ func (e *Engine) generateNotifications(o protocols.Objective) (EngineEvent, erro
 			}
 			outgoing.PaymentChannelUpdates = append(outgoing.PaymentChannelUpdates, info)
 		case *channel.Channel:
-			l, err := query.ConstructLedgerInfoFromChannel(c)
+			l, err := query.ConstructLedgerInfoFromChannel(c, *e.store.GetAddress())
```

Comment:
Out of scope: why does this store method return a pointer? I don't see the point (no pun intended).

Reply:
Unknown.